python/GangaDirac/Dockerfile

```diff
@@ -1,34 +1,28 @@
-FROM alexanderrichards/dirac_ui:latest
+# FROM taking build arg requires at least
+#   Docker version 17.05.0-ce, build 89658be
+ARG ganga_version=latest
+FROM ganga-core:$ganga_version
 LABEL maintainer "Alexander Richards <a.richards@imperial.ac.uk>"
-ARG ganga_version=6.5.2
+ARG dirac_version=v6r17p18
```

Comment:
Is it possible to put some help text here that explains it is essential to mount the directory containing the Grid proxy.